pkgs/development/interpreters/python/hooks/sphinx-hook.sh

```diff
@@ -35,20 +29,41 @@ buildSphinxPhase() {
         echo 2>&1 "Sphinx documentation not found, use 'sphinxRoot' variable"
         exit 1
     fi
-    sphinx-build -M html "${__sphinxRoot}" ".sphinx/html" -v
+
+    if [ -n "${sphinxBuilders-}" ]; then
+        eval "__sphinxBuilders=($sphinxBuilders)"
+    else
+        __sphinxBuilders=(html)
+    fi
+
+    for __builder in "${__sphinxBuilders[@]}"; do
+        echo "Executing sphinx-build with ${__builder} builder"
+        sphinx-build -M "${__builder}" "${__sphinxRoot}" ".sphinx/${__builder}" -v
+    done
 
     runHook postBuildSphinx
 }
 
 installSphinxPhase() {
+    echo "Executing installSphinxPhase"
+
     local docdir=""
     runHook preInstallSphinx
 
-    docdir="${doc:-$out}/share/doc/${sphinxOutdir:-$name}"
-    mkdir -p "$docdir"
+    for __builder in "${__sphinxBuilders[@]}"; do
+        # shellcheck disable=2154
+        docdir="${doc:-$out}/share/doc/${pname}"
+
+        # divert output for man builder
+        if [ "$__builder" == "man" ]; then
+            docdir="${man:-$out}/share/man/man1/"
```

Comment:
hm, I am not that much knowledgeable on manpages from Python software, but are we certain those are always in `man1` ?

Reply:
we can let it as-is and add a knob if needed at all later I assume

Reply:
That's a good point, but I believe I'm at the mercy of what sphinx provides here. Upstream needs to define what sections they document.

Reply:
Do the man pages have a .[1-9] suffix? If so we could use installManPage.